webapp/channels/src/components/post/post_component.tsx

```diff
@@ -624,7 +624,7 @@ const PostComponent = (props: Props): JSX.Element => {
                             {!props.isPostBeingEdited &&
                             <PostOptions
                                 {...props}
-                                teamId={teamId}
+                                teamId={teamId || ''}
```

Comment:
Minor nit, I'd like to see this done where `teamId` is set above since I'd usually expect a variable with that name to never be undefined